src/Illuminate/Routing/Middleware/ValidateSignature.php

```diff
@@ -18,10 +19,14 @@ class ValidateSignature
      */
     public function handle($request, Closure $next)
     {
-        if ($request->hasValidSignature()) {
+        if (URL::hasValidSignature($request)) {
```

Comment:
Why did you make this change?

Reply:
Because I think that direct call to `URL` facade is better than macro call with the exact same facade call under it.
Don't you think so?